xml/cap_depl_upgrade.xml

```diff
@@ -93,5 +149,32 @@ suse/uaa                     2.13.3  A Helm chart for SUSE UAA</screen>
 
 &prompt.user;helm upgrade --recreate-pods <replaceable>susecf-console</replaceable> suse/console \
  --values scf-config-values.yaml</screen>
+ 
+ <para>
+     Your pods will be unavailable to answer requests until the upgrade is
+     complete.
+     </para>
  </sect1>
+ 
+ <sect1 xml:id="sec.cap.skipped-release">
+     <title>Installing Skipped Releases</title>
+     <para>     
+By default, &helm; always installs the latest release. What if you accidentally 
+skipped a release, and need to apply it before upgrading to the current release? 
+Install the missing release by specifying the &helm; chart version number. For 
+example, your current UAA and SCF versions are 2.10.1. Consult the table at the 
+beginning of this chapter to see which releases you have missed. In this example, 
+the missing &helm; chart version for UAA and SCF is 2.11.0. Use the 
+<command>--version</command> option to install a specific version:
+</para>
+
+<screen>&prompt.user;helm install --version 2.11.0 suse/uaa \
```

Comment:
Given that we are in an upgrade process, should this not be `helm upgrade ...`, instead of `helm install` ?
See the box after `run the following commands to perform the upgrade` up in the document
(line 120+ of the xml ?) for comparison

Reply:
You are correct, thanks for catching this. Can't believe I missed it :man_facepalming: